WORKSPACE

```diff
@@ -103,7 +103,10 @@ gcs_file(
 
 load("@bazel_toolchains//rules:rbe_repo.bzl", "rbe_autoconfig", "rbe_autoconfig_root")
 
-rbe_autoconfig(name = "rbe_default")
+rbe_autoconfig(
+    name = "rbe_default",
+    force_checked_in_confs = True,
```

Comment:
What about merging two flags: "force_checked_in_confs" and "use_checked_in_confs" to be just one flag, maybe "use_checked_in_confs", and the enum value can be "force" (error out if no checked in configs are available), "attempt" (try finding checked-in configs and fall back to generate configs on the fly), and "false" (always generate the configs on the fly).

Reply:
done